### DISCUSSION TITLE
You said to say if we wanted a Typescript version...

### DISCUSSION THREAD

**Discussion by kevmcdonk** [microsoft/semantic-kernel#1207] [General]
So here is a thread on why people would want a Typescript version that others can add to

**Comment by kevmcdonk** [comment]
Now, I am new to Semantic Kernel and still learning. However, it seems excessive to need to have a Typescript based Bot using the new Teams AI that still requires a C# based Azure Function to call the Graph. Surely this could be built in as functions directly into the main model? That would greatly simplify deployments and acceptance from people.

**Comment by evchaki** [comment]
We don't have any current plans to support Typescript as a kernel.
